src/backend/distributed/utils/citus_safe_lib.c

```diff
@@ -295,3 +295,50 @@ SafeSnprintf(char *restrict buffer, rsize_t bufsz, const char *restrict format,
 	va_end(args);
 	return result;
 }
+
+
+/*
+ * SafeStringToInt32 converts a string containing a number to a int32. When it
+ * fails it calls ereport.
+ *
+ * The different error cases are inspired by
+ * https://stackoverflow.com/a/26083517/2570866
+ */
+int32
```

Comment:
nitpick: Can we move this after line 110 right after existing SafeStringToInt64.

Reply:
done